tests/Builder/BuilderEncoderTest.php

```diff
@@ -153,9 +154,8 @@ public function testPerformCount(): void
      *
      * @param list<int>          $limit
      * @param array<string, int> $expectedLimit
-     *
-     * @dataProvider provideExpressionFilterLimit
```

Comment:
We've excluded the `ForbiddenAnnotations` sniff due to a different issue, but if you'd like we can try to re-enable it and add all of the old PHPUnit annotations to the list. Not sure if PHPUnit 10 still supports annotations, but if it does it may be worth doing to make sure muscle memory doesn't kick in when writing tests.

Reply:
PHPUnit 10 still support annotations. But the rector rule will force usage of attributes.